values.yaml

```diff
@@ -749,7 +749,7 @@ registry:
   #   command: [ 'sh', '-c', "sleep 20" ]
   # Secret is used to secure the upload state from client
   # and registry storage backend.
-  # See: https://github.com/distribution/distribution/blob/main/docs/configuration.md#http
+  # See: https://github.com/distribution/distribution/blob/main/docs/content/about/configuration.md#http
```

Comment:
since Harbor is currently using distribution 2.8, maybe changing the URL to:
* https://github.com/distribution/distribution/blob/release/2.8/docs/configuration.md#http